screwdriver/screwdriver.go

```diff
@@ -41,11 +42,39 @@ type Step struct {
 	Command string `json:"command"`
 }
 
+// PairSlice is an associative array which preserves order
+type PairSlice struct {
```

Comment:
[IMO]
It seems to be easier to understand without the Body.
How about this?
```
type Environments []struct{
    Key string
    Value string
}
```
or 
```
type Pair struct {
    Key string
    Value string
}
type Environments []Pair
```

Reply:
In addition, `Environments` looks better name.

Reply:
I fixed it.